backend/core/serializers/user_settings.py

```diff
@@ -0,0 +1,34 @@
+from django.utils.translation import gettext_lazy as _
+from rest_framework import serializers
+from rest_framework.serializers import ValidationError
+
+
+class GenericPollUserSettingsSerializer(serializers.Serializer):
+    """
+    The settings common to all polls.
+    """
+
+    rate_later__auto_remove = serializers.IntegerField(required=False)
+
+    def validate_rate_later__auto_remove(self, value):
+        if value < 2:
```

Comment:
I'm not 100 % sure. We don't want negative value, or zero for sure, then I proposed to use `< 2` mainly because:
- it's easy to revert in case of need
- to motivate users to compare a video more than once
- our algorithms "prefer" having more than one input per user per video, right?
- to "educate" users about how the algorithms work 

I'm not opposed to allow `value = 1`, but as I didn't see any use case that would benefit from it I thought it would be better to keep the minimum value to `2`.

Do you think it's possible that some users will be frustrated to not be able to set the value to `1`?

Any thought @aidanjungo @amatissart @lfaucon ?

Reply:
Finally I changed my mind. I found one use case where it's convenient to allow users to set the setting to `1`: they have compared a lot of videos 2 times, they are not in their rate-later list anymore, and they decided to compare them once more. To do this they need to add the compared videos to the rate-later list again, and to set this setting to `1`.

I'm going to update the PR.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1297/commits/fffb3b78acc9820bcda476382d91c52400a2ef55.

Reply:
> Finally I changed my mind. I found one use case where it's convenient to allow users to set the setting to `1`: they have compared a lot of videos 2 times, they are not in their rate-later list anymore, and they decided to compare them once more. To do this they need to add the compared videos to the rate-later list again, and to set this setting to `1`.
> 
> I'm going to update the PR.

I think you misunderstand how it currently works. If I have compared a video 10 times and have configured removal after 4 comparisons, then I add the video to my rate later list, then it will be removed after a single comparison (because 11 >= 4)

Reply:
I agree we prefer >= 2, 1 is possible. I would propose as a best option that we allow 1, but we use the user setting form on the frontend to add a bit of explaination about how to choose this parameter and teach the users what to do with it. Maybe that would even lead contributors to care more than before about the number of comparison per video